arbitrum/multigas/resources_test.go

```diff
@@ -6,22 +6,24 @@ import (
 )
 
 func TestMultiGas(t *testing.T) {
-	t.Run("Test constructor", func(t *testing.T) {
-		// Test ZeroGas
+	t.Run("TestZeroGas", func(t *testing.T) {
```

Comment:
I've seen this pattern a few times in test code, and I really don't understand it. (Not saying it's wrong. I'm genuinely trying to understand if there is an advantage to structuring the test code this way.)

But, why do you prefer having all of these `t.Run("TestNameThatLooksLikeAFunction", func())` calls inside a parent test function, rather than just having separate test functions for each case?

```go
func TestZeroGas(t *testing.T) {
}

func TestComputationGas(t *testing.t) {
}
...
```

Reply:
This one is really just a discussion point, and not a change I need.

Reply:
I don't have a strong reason. In this case, it evolved to be that way, but it could easily be separate functions.